tests/infra/remote.py

```diff
@@ -350,6 +356,15 @@ def wait_for_stdout_line(self, line, timeout):
         )
 
 
+CCF_TO_OE_LOG_LEVEL = {
```

Comment:
I take it that if `--log-level=info`, the we get as many logs as we do now? Also, how verbose is OE's `INFO`?

Reply:
Yes, OE's INFO is very verbose, it's closer to our DEBUG.